src/load.ts

```diff
@@ -89,9 +140,28 @@ export default async function load(
     } else {
       // load remote
       const headers = new Headers();
+
+      // Add custom parsed HTTP headers
+      if (options.httpHeaders) {
+        const parsedHeaders = parseHttpHeaders(options.httpHeaders);
+        Object.keys(parsedHeaders).forEach((headerKey) => {
+          const customVal = parsedHeaders[headerKey];
+          headers.set(headerKey, customVal as string);
+        });
+      }
+
+      // Default OpenAPITypescript header overrides
       headers.set("User-Agent", "openapi-typescript");
-      if (options.auth) headers.set("Authorization", options.auth);
-      const res = await fetch(schemaID, { method: "GET", headers });
+      if (options.auth) {
+        headers.set("Authorization", options.auth);
+      }
+
+      // Fetch Swagger Schema by schemaID in cache/URL via GET request
+      let httpMethod: HTTPVerb = "GET";
+      if (options.httpMethod && typeof options.httpMethod === "string" && isValidHTTPMethod(httpMethod)) {
```

Comment:
`typeof undefined !== "string "` so let's simplify this condition a bit:
```suggestion
      if (typeof options.httpMethod === "string" && isValidHTTPMethod(httpMethod)) {
```

Reply:
Fair enough, this was an oversight on my end, thank you for pointing this out.￼

Reply:
Complete